src/tcptunnel.h

```diff
@@ -13,6 +13,7 @@
 #define STAY_ALIVE_OPTION   'h'
 #define HELP_OPTION         'i'
 #define VERSION_OPTION      'j'
+#define	CLIENT_ADDR_OPTION  'k'
```

Comment:
Can you move the `CLIENT_ADDRESS_OPTION` line under the `BIND_ADDRESS_OPTION` line? So it matches the options order.